playbooks/roles/jenkins/tasks/jobs.yml

```diff
@@ -5,10 +5,7 @@
 
 - name: copy a requirements file with pinned Python dependencies
   copy:
-    content: |
-      python-jenkins==0.4.15  # newer ones have broken basic auth :(
-      jenkins-job-builder==1.6.2
-      git+https://github.com/rusty-dev/jenkins-job-builder-pipeline.git@f24b9f5b0de03edd59b94061fa5182d11887403d#egg=jenkins_job_builder_pipeline
+    src: ../../../../requirements-jenkins-job-builder.txt
```

Comment:
Is there a way we can just consolidate these with the ansible depoendency in requirements.txt and install them all for tests?

Ansible is a requirement of this package but we don't test it anywhere. That doesn't mean it shouldn't be installed when we're running tests. It could have conflicting dependencies. with one of the packages here.